src/lib/registration/clients/online_search.rb

```diff
@@ -77,13 +77,15 @@ def workflow_aliases
 
       # Runs the sequence
       #
-      # This method performs these steps:
+      # This sequence performs the following steps:
       #
-      #   1. Find the addons
-      #   2. Search for packages (UI)
-      #   3. Register the addons
-      #   4. Display the EULA if needed
-      #   5. Select the packages for installation
+      #   1. Check whether the online search is supported in the underlying system
+      #   2. Find the addons
```

Comment:
I think this is not clear enough. Maybe 

```suggestion
      #   2. Get the available addons from the registration server
```

is better?